### ISSUE TITLE
Vulnerabilities in depdendencies

### ISSUE THREAD

**Issue by ReekenX** [ngryman/cz-emoji#37]
In recent `cz-emoji` version:

```
$ grep cz-emoji package.json
23:        "cz-emoji": "^1.2.1",
```

`yarn audit` reports vulnerabilities:

```
$ yarn audit
[..]
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ low           │ Denial of Service                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ mem                                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=4.0.0                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ cz-emoji                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ cz-emoji > inquirer-maxlength-input-prompt > jest > jest-cli │
│               │ > yargs > os-locale > mem                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/1084                      │
└───────────────┴──────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ low           │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ minimist                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=0.2.1 <1.0.0 || >=1.2.3                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ cz-emoji                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ cz-emoji > inquirer-maxlength-input-prompt > jest > jest-cli │
│               │ > istanbul-api > istanbul-reports > handlebars > optimist >  │
│               │ minimist                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/1179                      │
└───────────────┴──────────────────────────────────────────────────────────────┘
````

It is low priority vulnerabilities, but it would be nice to have no vulnerabilities at all since we use this package in multiple projects.

Maybe version bump of `inquirer-maxlength-input-prompt` could solve this?

**Comment by dialex** [comment]
+1
Can you please upgrade all your dependencies to address all this security issues? My project is using `cz-emoji` and currently I have 66 security issues, 100% related with your repo :( 

If you can't fix this, we will have to stop using your repo

**Comment by dialex** [comment]
Ok, I've forked this repo to investigate the matter. Apparently, the issue is not with this repo, but with this dependency: https://github.com/jwarby/inquirer-maxlength-input-prompt/issues/1

The project is inactive for the last 2 years and their Jest version uses an insecure version of `mem`. We need to pressure them to upgrade their dependencies, not this repo.

**Comment by ReekenX** [comment]
@dialex created a PR on `inquirer-maxlength-input-promp` to fix security issue. But it has been 3 months now and there is no author's response nor any activity for years. I assume that project is no longer maintained.

Could we expect dependency removed @ngryman ? Every project using this is getting reports about cz-emoji vulnerabilities for past 4 months now.

**Comment by ReekenX** [comment]
Author of `inquirer-maxlength-input-prompt` merged security fix PR, could you @ngryman bump this project version now? This should resolve this issue.

**Comment by ReekenX** [comment]
Ping @ngryman

**Comment by ngryman** [comment]
Hey folks, sorry for the late response and thanks for your patience 🙏 
I just released [v1.2.2](https://github.com/ngryman/cz-emoji/releases/tag/v1.2.2) which bumps `inquirer-maxlength-input-prompt`.

This project is still maintained. However, I have a pretty limited bandwidth currently and unfortunately not enough time to allocate to this specific project. I could definitely use help to maintain it. Let me know if you are interested in becoming contributors.
